### ISSUE TITLE
What is the current status of the lib

### ISSUE THREAD

**Issue by mikart143** [andlabs/libui#510]
Hello!
Firstly, I do not want to put pressure on the author.
I am planning to use this lib in my rather big academic project.
I just want to know if this is going to be maintained ?
I checked remodel branch and frequency of commits, everything looks good up to 8th Jun.
Also the patreon issue is not resolved which gives us some point of view that author lost his interest in the project.

Mike

**Comment by Noxware** [comment]
Looks pretty... incomplete for me... I don't know how I ended here but there is a library called [Yue](https://libyue.com), is native, lightweight, has clipboard support, tray support, native webview support, native dialogs support, easy transparent and floating window support (check the floating heart example in its github repo)... it makes me very sad that yue is not very known...

**Comment by quinnjr** [comment]
Check out the insights for the project:

https://github.com/andlabs/libui/pulse/monthly

**Comment by joa-quim** [comment]
Yes, but looking at https://github.com/andlabs/libui/graphs/code-frequency is not encouraging at all.

**Comment by marcotrosi** [comment]
> Yes, but looking at https://github.com/andlabs/libui/graphs/code-frequency is not encouraging at all.

but why do you care then? simply don't use libui. problem solved. 😀

**Comment by terremoth** [comment]
I would love to see this lib stable, and I would use it in my projects.
Also: I am one that would be glad to contribute via patreon for the autor and people to continue it.

**Comment by rubyFeedback** [comment]
To marcotrosi - libui fills a great niche. If people can just use it easily,
even on windows, for a simple GUI, then this is GREAT. Getting
ruby-gtk available, for example, is significantly harder; even on WSL
on windows 10. I can do so, but imagine a +70 years old person who
never had such an exposure before. This is why a simple GUI can
be sooooooo helpful - so I totally disagree with your comment.

As for alternatives, well - competition is good. One will always 
rise to be better, so people can rally around that (or fork if it
becomes stale).
